root/defaults/default.incontainer

```diff
@@ -0,0 +1,27 @@
+server {
+	listen 80 default_server;
+	root /var/www/html;
+	index index.html index.htm index.php;
+
+	server_name _;
+
+#	ssl_certificate /config/keys/cert.crt;
+#	ssl_certificate_key /config/keys/cert.key;
+
+	client_max_body_size 0;
+
+	location / {
+		try_files $uri $uri/ /index.html /index.php?$args =404;
+	}
+
+	location ~ \.php$ {
+		fastcgi_split_path_info ^(.+\.php)(/.+)$;
+		# With php5-cgi alone:
```

Comment:
These commented lines can be removed as they are old, unnecessary and outdated (php7 now instead of 5)